classes/person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative './nameable'
+
 class Person
```

Comment:
- [ ] It would be better if our Person class inherits from the Nameable class

Reply:
resolved